sdk/go/common/apitype/service.go

```diff
@@ -33,6 +33,11 @@ const (
 
 	// Indicates that the service backend supports batch encryption.
 	BatchEncrypt APICapability = "batch-encrypt"
+
+	// Indicates whether the service supports a notion of providing an opinion on a
+	// default organization among the user's org memberships, if a default org has
+	// not been explicitly defined.
+	DefaultOrganization APICapability = "default-org"
```

Comment:
It upsets me that we have three names for the same thing here.
"default-org"
"DefaultOrganization"
"DefaultOrgs"

Can we name all these the same, probably DefaultOrg

Reply:
lol oops, can you tell I wrote this in between three separate sessions 🙈